specification/compute/resource-manager/readme.md

```diff
@@ -151,6 +151,21 @@ directive:
       - TrackedResourceGetOperation
       
 ```
+
+### Tag: package-2018-06

```

Comment:
Please also add a `package-gallery-2018-06` with only gallery. Tag with consistent ApiVersion are required for Go, Python and Ruby

Reply:
I am not sure if I modified it correctly. Would you mind taking a look?